tests/features/api.feature

```diff
@@ -0,0 +1,283 @@
+        | Featured media legend   | Event media legend                                            |
+        | Summary for report      | Event summary for report                                      |
+        | Report text             | Report text                                                   |
+        | Registration start date | 2020-03-01 12:30:00                                           |
+        | Registration end date   | 2020-03-10 18:30:00                                           |
+        | Start date              | 2020-06-15 12:30:00                                           |
+        | End date                | 2020-06-20 18:30:00                                           |
+        | Registration URL        | http://example.com                                            |
+        | Registration capacity   | Event capacity                                                |
+        | Entrance fee            | 1234                                                          |
+        | Online type             | facebook                                                      |
+        | Online time start       | 2019-02-21 09:15:00                                           |
+        | Online time end         | 2019-02-21 14:00:00                                           |
+        | Online description      | Event online description                                      |
+        | Online link             | uri: http://ec.europa.eu/info - title: Info site              |
+        | Languages               | Valencian                                                     |
+        | Organiser is internal   | Yes                                                           |
+        | Internal organiser      | Directorate-General for Informatics                           |
+        | Status                  | as_planned                                                    |
+        | Event website           | uri: https://ec.europa.eu/inea/ - title: INEA                 |
+        | Social media links      | uri: http://example.com - title: Twitter - link_type: twitter |
+
+      When I visit node "My event" edit page
+      Then I should see the text "My event"
+      And I should see the text "Exhibitions"
+      And I should see the text "Short intro"
+      And I should see the text "Event description summary"
+      And I should see the text "Event description"
+      And the "Featured media legend" field should contain "Event media legend"
+      And I press "Event report"
+      And I should see the text "Event summary for report"
+      And I should see the text "Report text"
+      And I press "Registration"
+      And I should see the text "http://example.com"
+      And the "Registration capacity" field should contain "Event capacity"
+      And the "Entrance fee" field should contain "1234"
+      And I should see the text "Facebook"
+      And the "oe_event_registration_dates[0][value][day]" field should contain "1"
```

Comment:
can't we use shouldBeSelectedForSelect in these situations ?

Reply:
I refactored the statements